compiler/z/codegen/S390Instruction.cpp

```diff
@@ -1269,6 +1269,87 @@ TR::S390PseudoInstruction::estimateBinaryLength(int32_t currentEstimate)
    return currentEstimate + estimate;
    }
 
+////////////////////////////////////////////////////////////////////////////////
+// TR::S390DebugCounterBumpInstruction:: member functions
+////////////////////////////////////////////////////////////////////////////////
+
+/**
+ * This instruction is bumps the value at the snippet corresponding to the address of the debug counter count.
+ * The valid opcode is TR::InstOpCode::DCB
+ *
+ * Either 2 or 4 instructions are encoded matching the following conditions:
+ * 
+ * If a free register was saved:                      Otherwise if a spill is required:
+ *                                                    STG     Rscrtch,   offToLongDispSlot(,GPR5)
+ * LGRL    Rscrtch,   counterRelocation               LGRL    Rscrtch,   counterRelocation       
+ * AGSI 0(Rscrtch),   delta                           AGSI 0(Rscrtch),   delta                   
+ *                                                    LG      Rscrtch,   offToLongDispSlot(,GPR5)
+ */
+uint8_t *
+TR::S390DebugCounterBumpInstruction::generateBinaryEncoding()
+   {
+   // *this    swipeable for debugging purposes
```

Comment:
We don't need these comments anymore. They are artifacts of ~10+ years ago so that the `this` pointer is available when running the code under a debugger such that the debugger can display the value.